react-toggle-switch-component/my-app/src/ToggleButton.jsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react';
+
+export default function ToggleButton({ label }) {
+  const [isOn, setIsOn] = useState(true);
+
+  let bgColor;
+  let position;
+
+  if (isOn) {
+    label = 'ON';
+    bgColor = 'green';
+    position = 1;
+  } else {
+    label = 'OFF ';
+    bgColor = 'red';
+    position = -31;
+  }
+
+  return (
+    <>
+      <button
+        onClick={() => setIsOn(!isOn)}
+        style={{ backgroundColor: bgColor }}
+        className="toggle-switch">
+        <button className="slider" style={{ translate: position }}></button>
```

Comment:
Button inside of button is awkward. There's not a good reason for this to be a button, since you don't have an `onClick` handler for it.

According to MDN, it's actually not legal to put a button inside another button. Do you get warnings in your console?

Reply:
I actually did not get any errors when I put the button inside another button. I don't know why I did not think of using a p element instead.